lib/bolt/cli.rb

```diff
@@ -563,7 +563,8 @@ def execute(options)
             when 'command'
               executor.run_command(targets, options[:object], executor_opts)
             when 'script'
-              script_path = find_file(options[:object])
+              script_path = find_file(options[:object], executor.future&.fetch('file_paths', false))
```

Comment:
We could update `Bolt::Config` to set `'future' => {}` by default so we can get rid of the safe operator.

Reply:
I thought about that, but I think at some point it might useful to know if `future` was set at all or not. We may not end up needing that, but I don't want to rule it out so early.